ajax/libs/icheck-bootstrap/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "icheck-bootstrap",
+  "description": "Pure css checkboxes and radio buttons for Twitter Bootstrap.",
+  "filename": "icheck-bootstrap.css",
```

```suggestion
  "filename": "icheck-bootstrap.min.css",
```

A minified version exists, so we should prefer to suggest that.